release/create-constraints.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+"""Parse versions from buildout.cfg and write them into constraints.txt.
+
+This script is called by 'tox -c release/tox.ini -e constraints'
+
+It only needs to work in Python 3.
+
+Manual usage: bin/python create-constraints.py
+zc.buildout needs to be importable in that python.
+"""
+from zc.buildout import buildout
+
+import os
+import sys
+
+
+# List packages that we should not include.
+# Originally this was for packages that are pinned by Zope
+# but cannot be pip installed on Py 2 or on Py 3.
+BLACKLIST = [
```

Comment:
Well, better `DENYLIST`?

Reply:
```suggestion
DENYLIST = [
```

Reply:
Ah, yes, same as in the hotfix. I will change it and then merge.